frontend/src/components/ExpensePage.js

```diff
@@ -1,21 +1,50 @@
-import React from "react";
+import React, { Component } from "react";
 import Header from "./Header";
 import ExpensePresentation from "./ExpensePresentation";
 import ExpenseList from "./ExpenseList";
 import Footer from "./Footer";
+
 import "../styles/ExpensePage.css";
 
-function ExpensePage() {
-  return (
-    <div className="ExpensePage">
-      <Header />
-      <main>
-        <ExpensePresentation />
-        <ExpenseList />
-      </main>
-      <Footer />
-    </div>
-  );
+export class ExpensePage extends Component {
+  state = {
+    total: 0,
+  };
+
+  updateTotal = (amount) => {
+    this.setState({ total: this.state.total + parseFloat(amount) });
```

Comment:
State should also be saved for the update (just to be safe we read the "correct" state):
``` JS
this.setState((state) => ({ total: state.total + parseFloat(amount) }));
```

Reply:
Need more clarity

Reply:
Made changes as requested.